Clean-up IO classes

HannesWell opened the pull request:
With this PR I suggest improvements for the classes in the `org.jzy3d.io` package. Many of the methods implemented there are provided by the Java standard library and can be used from there.

I've deprecated the methods that are obsolete. IIRC jzy3d-2.0.0 was not yet released so API changes are possible, so these methods could be removed immediately? If you agree with these changes I can remove the deprecated methods.

HannesWell commented:
For me it looks like the failure is not related to this change.

jzy3d commented:
Thanks for the cleanup, this package is very old (and little awfull).

It seams Path.of method does not exist in Java 1.8. Maybe your local maven project is using a more recent JDK? Sticking to an old Java version is made on purpose to remain compatible with a maximum number of software.

HannesWell commented:
Your welcome. I got the same impression 😅
Yes `Path.of` was introduce in Java-11 (which I have installed on my computer). I've already changed all occurrences to `Paths.get` which is available in Java-8. The same applies for `Files.writeString()` which I have also replaced already.

Furthermore it looks like the entire package is not used in library. Unless you want to provide it as utility for consumers it might be a candidate for removal.

At the moment the test failed in `org.jzy3d.emulgl.trials.TestContinuousAndOnDemandRendering.whenViewShoot_thenCameraIsProperlySet()` from `Jzy3d Core AWT` which is executed after jzy3d-core was build.

HannesWell commented:
Btw. if I see it right, jzy3d requires Java-1.8? But your build here at github uses Java-9.
